src/Components/Common/FilePreviewDialog.tsx

```diff
@@ -124,7 +157,17 @@ const FilePreviewDialog = (props: FilePreviewProps) => {
               <Cancel onClick={onClose} label="Close" />
             </div>
           </div>
-          <div className="flex flex-1 items-center justify-center">
+          <div className="flex flex-1 items-center justify-center gap-2">
+            <ButtonV2
+              className={
+                uploadedFiles && index === 0
+                  ? "pointer-events-none invisible opacity-0"
+                  : ""
+              }
```

Comment:
To hide the button, you can simply use `{(condition) && <ButtonV2>...</ButtonV2>}`.
To disable the button, you can simply set the `disabled` prop to `true`.